branded-checkout.html

```diff
@@ -11,7 +11,7 @@
 </head>
 <body>
 
-<branded-checkout designation-number="2294554" show-cover-fees="true" use-v3="true" ng-cloak></branded-checkout>
+<branded-checkout designation-number="2294554" show-cover-fees="true" use-v3="true" hide-spouse-details="true" ng-cloak></branded-checkout>
```

Comment:
This commit (`Allow branded checkout to configure hiding spouse information initially, and show it when button is clicked.`) is where you should rebase the squash for branded-checkout.html into so it never happened.